plotsquared/api/api-documentation.md

```diff
@@ -0,0 +1,161 @@
+    compileOnly("com.plotsquared:PlotSquared-Core")
+    compileOnly("com.plotsquared:PlotSquared-Bukkit") { isTransitive = false }
+}
+```
+
+### Maven - PlotSquared Core
+
+```xml
+<repositories>
+    <repository>
+        <id>papermc</id>
+        <url>https://repo.papermc.io/repository/maven-public/</url>
+    </repository>
+</repositories>
+<dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>com.intellectualsites.bom</groupId>
+            <artifactId>bom-newest</artifactId>
+            <version>1.32</version>
+            <scope>import</scope>
+            <type>pom</type>
+        </dependency>
+    </dependencies>
+</dependencyManagement>
+<dependencies>
+    <dependency>
+        <groupId>com.plotsquared</groupId>
+        <artifactId>PlotSquared-Core</artifactId>
+        <scope>provided</scope>
+    </dependency>
+</dependencies>
+```
+
+### Maven - PlotSquared Core and Bukkit
+
+```xml
+<repositories>
+    <repository>
+        <id>papermc</id>
+        <url>https://repo.papermc.io/repository/maven-public/</url>
+    </repository>
+</repositories>
+<dependencyManagement>
+<dependencies>
+    <dependency>
+        <groupId>com.intellectualsites.bom</groupId>
+        <artifactId>bom-1.18.x</artifactId>
+        <version>1.32</version>
+        <scope>import</scope>
+        <type>pom</type>
+    </dependency>
+</dependencies>
+</dependencyManagement>
+<dependencies>
+<dependency>
+    <groupId>com.plotsquared</groupId>
+    <artifactId>PlotSquared-Core</artifactId>
+    <scope>provided</scope>
+</dependency>
+
+<dependency>
+    <groupId>com.plotsquared</groupId>
+    <artifactId>PlotSquared-Bukkit</artifactId>
+    <scope>provided</scope>
+    <exclusions>
+        <exclusion>
+            <artifactId>PlotSquared-Core</artifactId>
+            <groupId>*</groupId>
+        </exclusion>
+    </exclusions>
+</dependency>
+</dependencies>
+```
+
+### Useful classes for PlotSquared
+
+* [PlotAPI](https://github.com/IntellectualSites/PlotSquared/blob/v6/Core/src/main/java/com/plotsquared/core/PlotAPI.java)
+* [PlotPlayer](https://github.com/IntellectualSites/PlotSquared/blob/v6/Core/src/main/java/com/plotsquared/core/player/PlotPlayer.java)
+* [FlagContainer](https://github.com/IntellectualSites/PlotSquared/blob/v6/Core/src/main/java/com/plotsquared/core/plot/flag/FlagContainer.java)
+* [SchematicHandler](https://github.com/IntellectualSites/PlotSquared/blob/v6/Core/src/main/java/com/plotsquared/core/util/SchematicHandler.java)
+* [ChunkManager](https://github.com/IntellectualSites/PlotSquared/blob/v6/Core/src/main/java/com/plotsquared/core/util/ChunkManager.java)
+* [UUIDPipeline](https://github.com/IntellectualSites/PlotSquared/blob/v6/Core/src/main/java/com/plotsquared/core/uuid/UUIDPipeline.java)
+
+## Tutorials
+
+* [Getting an instance of PlotSquared](event-api.md#_getting_an_instance)
+* [Flag API](flag-api.md)
+* [Event API](event-api.md)
```

Comment:
Clicking on the hyperlink should link within the document on gitbook. Currently, it links to GitHub.